masq_lib/Cargo.toml

```diff
@@ -11,7 +11,7 @@ workspace = "../node"
 [dependencies]
 actix = "0.7.9"
 clap = "2.33.1"
-crossbeam-channel = "0.4"
+crossbeam-channel = "0.5.0"
```

Comment:
Are we updating crossbeam everywhere, or just here? If just here, update the other `Cargo.toml`s too.

Reply:
Here, it was actually behind the other versions that we can see in other files. So by now all are the same.